docusaurus/docs/_cli-log-levels-and-formats.md

```diff
@@ -2,7 +2,7 @@
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-All Ziti components log to standard output and standard error file descriptors.
+All Go Ziti components log to standard output and standard error file descriptors, so their output appears in the systemd journal (`journalctl -xeu ziti-controller.service`) when installed from a Linux package, and the Docker log (`docker compose logs ziti-router`) when run as a container. The C/C++ `ziti-edge-tunnel` has distinct log levels and controls, described in [Linux tunneler troubleshooting](/reference/tunnelers/60-linux/50-linux-tunnel-troubleshooting.mdx).
```

Comment:
I'd like to remove "Go ". It doesn't add any usuable information to the reader to know that these are "go" binaries and is something that might end up being rendered obsolete someday in the future which makes me dislike such a seemingly small addition. I'd like to return to the original wording. 

The remaining information was once 'reference' doc but here you've moved it to the guides section. This makes it feel wrong when reading it. It no longer feels relevant to logging in linux.

This section should focus on:
* You will log with json
* You will log to journald
* here's how you ship logs from the controller to some log collection agent

This is a guide. it should tell you how to do it and not present different options